pkg/build/controller/build/defaults/defaults_test.go

```diff
@@ -155,28 +155,82 @@ func TestGlobalProxyDefaults(t *testing.T) {
 	}
 
 	admitter := BuildDefaults{nil, defaultsProxy}
+
+	// source builds should have the defaulted env vars applied to the build pod
 	pod := testutil.Pod().WithBuild(t, testutil.Build().WithSourceStrategy().AsBuild())
 	err := admitter.ApplyDefaults((*corev1.Pod)(pod))
 	if err != nil {
 		t.Fatalf("unexpected error: %v", err)
 	}
-	build, err := common.GetBuildFromPod((*corev1.Pod)(pod))
```

Comment:
do the proxy env vars get set on the build itself or not?  (ie are they set in the context of the Dockerfile and s2i assemble execution?  do they end up set on the output application image?)

I would think they should *not* be, but i don't see anything asserting the behaviour either way.

Reply:
They do not get set on the build, I will add some code to this test that verifies that.

Reply:
excellent, though it looks like the previous incarnation of this test was expecting that they *were* set on the build?

Reply:
that was a mistake ....